recipes/rnabob/removeman.patch

```diff
@@ -0,0 +1,10 @@
+--- Makefile	2016-09-18 01:24:06.664551971 +0200
++++ Makefile-new	2016-09-18 02:00:43.872926057 +0200
+@@ -58,7 +58,6 @@
+ 
+ install: $(PROGS)
+ 	cp $(PROGS)   $(BINDIR)/
+-	cp rnabob.man $(MANDIR)/man1/rnabob.1
```

Comment:
I though I can only give a value to ```$PREFIX``` when it is actually used in the Makefile. Am I wrong?